com.google.EarthPro.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: com.google.EarthPro
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: earth-runner
+separate-locales: false
+finish-args:
+  - --env=PATH=/usr/bin:/app/bin
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --persist=.googleearth
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: earth
+    buildsystem: simple
+    build-commands:
+      - mv earth/* /app/
+      - install -m755 earth-runner /app/bin/earth-runner
+      - install -Dm644 /app/google/earth/pro/google-earth-pro.desktop /app/share/applications/com.google.EarthPro.desktop
+      - install -Dm644 /app/google/earth/pro/product_logo_256.png /app/share/icons/hicolor/256x256/apps/com.google.EarthPro.png
+      - install -Dm644 /app/google/earth/pro/product_logo_128.png /app/share/icons/hicolor/128x128/apps/com.google.EarthPro.png
+      - install -Dm644 /app/google/earth/pro/product_logo_64.png /app/share/icons/hicolor/64x64/apps/com.google.EarthPro.png
+      - install -Dm644 /app/google/earth/pro/product_logo_48.png /app/share/icons/hicolor/48x48/apps/com.google.EarthPro.png
+      - install -Dm644 /app/google/earth/pro/product_logo_32.png /app/share/icons/hicolor/32x32/apps/com.google.EarthPro.png
+      - install -Dm644 /app/google/earth/pro/product_logo_16.png /app/share/icons/hicolor/16x16/apps/com.google.EarthPro.png
+      - install -Dm644 com.google.EarthPro.xml /app/share/metainfo/com.google.EarthPro.metainfo.xml
+      - desktop-file-edit --remove-key=MultipleArgs /app/share/applications/com.google.EarthPro.desktop
+      - desktop-file-edit --set-icon=com.google.EarthPro /app/share/applications/com.google.EarthPro.desktop
+      - desktop-file-edit --set-key=Exec --set-value='earth-runner' /app/share/applications/com.google.EarthPro.desktop
+    build-options:
+      no-debuginfo: true
+    sources:
+      - type: archive
+        url: https://dl.google.com/dl/linux/direct/google-earth-pro-stable-7.3.6.x86_64.rpm
+        sha256: 91a4ab1e7a01f8582f83238164fd5943b3114c64505589904e402538acffa86e
+        dest: earth
+        x-checker-data:
+          type: html
+          url: https://support.google.com/earth/answer/168344
+          pattern: <li><a href="(https://dl.google.com/dl/linux/direct/google-earth-pro-stable-([\d.]+\d).x86_64.rpm)">
+      - type: file
+        path: com.google.EarthPro.xml
+      - type: script
+        dest-filename: earth-runner
+        commands:
+          - exec /app/google/earth/pro/googleearth
+
+  - name: libglu
```

Comment:
this should be build before because it's a dependency, even if it doesn't matter in practice.

Reply:
moved it